keys_for_testing.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+cwd=$(pwd)
+export MICROSETTA_INTERFACE_DEBUG_JWT_PUB=${cwd}/jwtRS256.key.pub
```

Comment:
Since you're disabling jwt verification, I don't understand the purpose of these key files.  The jwt is made up of three separate base64 strings separated by periods.  The first one specifies the algorithm, the second is the claims, the third is the signature.  We should literally be able to base 64 some claims json and put periods on either end and it should pass with verification disabled.  

I just get worried seeing private keys anywhere.

Reply:
-interface decodes the keys. We could also allow disabling that, instead of generating keys, but generating keys is pretty easy.

Reply:
Agree re: private keys, but these are burner one-use-only, and unless a future commit changes this, not dumped to stdout anywhere